src/domains/Townhall/TownHallHistory/TownHallHistory.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Divider from '@material-ui/core/Divider';
+import { ListItemAvatar, Avatar } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
```

Comment:
Was this to get around typescript?  Just looked up its use case because I'd never seen it before.  I'm curious on if typescript was giving you an error and what it was.

Reply:
No this was styling used in the Material UI List Component